build/mirror-repo.yml

```
@@ -0,0 +1,19 @@
+name: Mirror-to-client-repo
+
+on:
+  pull_request:
```

```suggestion
  push:
```
So it only runs when merged or pushed to development, instead of PR creation